docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -257,34 +333,35 @@ sql>
 +
 [source,sql]
 ----
-CREATE MAPPING cities (
-__key INT, 
-countries VARCHAR, 
-cities VARCHAR)
-type IMap OPTIONS('keyFormat'='int', 'valueFormat'='json-flat');
+CREATE OR REPLACE MAPPING cities (
+__key VARCHAR, 
+country VARCHAR, 
+city VARCHAR,
+population INT)
+type IMap OPTIONS('keyFormat'='varchar', 'valueFormat'='json-flat');
 ----
 
-. Add some countries and their capital cities to the map.
+. Add some data to the map.
 +
 [source,sql]
 ----
 INSERT INTO cities VALUES
-(1, 'United Kingdom','London'),
-(2, 'United Kingdom','Manchester'),
-(3, 'United States', 'New York'),
-(4, 'United States', 'Los Angeles'),
-(5, 'Turkey', 'Ankara'),
-(6, 'Turkey', 'Istanbul'),
-(7, 'Brazil', 'Sao Paulo'),
-(8, 'Brazil', 'Rio de Janeiro');
+(1, 'United Kingdom','London', 9_540_576),
+(2, 'United Kingdom','Manchester', 2_770_434),
+(3, 'United States', 'New York', 19_223_191),
+(4, 'United States', 'Los Angeles', 3_985_520),
+(5, 'Turkey', 'Ankara', 5_309_690),
+(6, 'Turkey', 'Istanbul', 15_636_243),
+(7, 'Brazil', 'Sao Paulo', 22_429_800),
+(8, 'Brazil', 'Rio de Janeiro', 13_634_274);
 ----
 --
 ====
 
 [[step-3]]
 == Step 3. Query the Cache with SQL
 
-Now that you have some data in your cluster, you can query it, using SQL.
+Now that you have some data in your cluster, you can query it, using SQL. If you're using the CLI, enter the following  queries in the SQL prompt. If you're using a client library, enter the following queries in the SQL Browser.
```

Comment:
Remove comma before ...using SQL?

Now that you have some data in your cluster, you can query it using SQL.